resources/definitions/nwa3d_a5.def.json

```
@@ -0,0 +1,65 @@
+{
+    "name": "NWA3D A5",
+    "version": 1,
```

Currently it has to be `"version": 2`
```suggestion
    "version": 2,
```